scan/spec/test_command_generator_spec.rb

```diff
@@ -5,6 +5,31 @@
     @project = FastlaneCore::Project.new(config)
   end
   before(:each) do
+    @valid_simulators = "== Devices ==
+-- iOS 9.3 --
+    iPhone 4s (238767C4-AF29-4485-878C-7011B98DCB87) (Shutdown)
```

Comment:
@jules2689 Yah, I'd love to use it, but this would mean we'd have to drop Xcode 6 support.